src/adapters/bidderFactory.js

```diff
@@ -176,10 +177,31 @@ export function newBidder(spec) {
       // After all the responses have come back, call done() and
       // register any required usersync pixels.
       const responses = [];
-      function afterAllResponses(bids) {
-        const bidsArray = bids ? (bids[0] ? bids : [bids]) : [];
+      let bidAcc = [];
+      function afterAllResponses() {
+        bidderRequest.end = timestamp();
+        let bidIds = bidAcc.map(bid => {
+          if (bid.requestId) {
+            return bid.requestId.toString();
+          }
+          return ' '
+        });
+
+        let missingBids = bidderRequest.bids.filter(bid => {
+          if (bid.bidId) {
+            return bidIds.indexOf(bid.bidId.toString()) === -1;
+          }
+          return false
+        });
 
-        const videoBid = bidsArray.some(bid => bid.mediaType === 'video');
+        missingBids.forEach(bid => {
+          const bidRequest = bidRequestMap[bid.bidId];
+          bid.requestId = bid.bidId;
+          let prebidNoBid = Object.assign(bidfactory.createBid(CONSTANTS.STATUS.NO_BID, bidRequest), bid);
```

Comment:
With Prebid 1.0 we are not adding NO_BID to auction. 
To close aucton in legacy we used to match bidsRequested and bidsReceived and hence were adding a NO_BID.

Not needed here.

Reply:
This is one of the main pieces to this ticket, so is definitely needed here. It’s adding back in “no bids” for all the missing analytics/data caused by the oversight of removing them in 1.*